data/src/main/java/com/linkedin/data/schema/IndentedPdlBuilder.java

```diff
@@ -133,4 +139,15 @@ private void writeLine(String code) throws IOException
   {
     indent().write(code).newline();
   }
+
+  private String getIndentSpaces(int indentDepth)
```

Comment:
I would make this method static, since it's just a helper method to compute indent text. Whether to use static methods when possible or to avoid static methods seems to be a contentious and subjective issue, so it's up to you tbh.

Reply:
lets keep it like this for now, as scope of method is for this class only